test/code_size/hello_wasm_worker_wasm.json

```diff
@@ -5,8 +5,8 @@
   "a.js.gz": 455,
   "a.ww.js": 115,
   "a.ww.js.gz": 127,
-  "a.wasm": 1850,
-  "a.wasm.gz": 1050,
-  "total": 3248,
-  "total_gz": 2016
+  "a.wasm": 1894,
```

Comment:
Why did this grow by 44 bytes?

Reply:
I think because it fixes the wasm workers implementation of errno_location so that it returns a TLS location, which I think is bug fix.

Reply:
Prior to this change the WW version of errno_location was the same as the non-threaded version (i.e. not thread local or thread safe).

Reply:
Ah, right... good fix then.